accord-core/src/main/java/accord/local/cfk/Updating.java

```diff
@@ -788,7 +788,13 @@ static CommandsForKeyUpdate updateUnmanaged(CommandsForKey cfk, SafeCommand safe
         if (safeCommand.current().hasBeen(Status.Truncated))
             return cfk;
 
-        Command.Committed command = safeCommand.current().asCommitted();
+        Command orig = safeCommand.current();
+        if (orig.saveStatus() == SaveStatus.Uninitialised)
```

Comment:
maybe static import, and inline the two conditions? Also, good to use `orig.txnId().is(EphemeralRead)`